tests/selftest.intern.js

```diff
@@ -16,6 +16,10 @@ define({
 
 	maxConcurrency: 2,
 	tunnel: 'BrowserStackTunnel',
+	tunnelOptions: {
```

Comment:
This information is always provided via command-line and shouldn’t be put in the test file